C7/Title_Screen.jpg.import

```diff
@@ -0,0 +1,35 @@
+[remap]
```

Comment:
StackOverflow says we should commit these: https://stackoverflow.com/questions/54249305/godot-what-are-import-files-and-should-you-commit-them-to-git